src/action.spec.ts

```diff
@@ -1,4 +1,3 @@
-import * as T from './type-helpers';
```

Comment:
Please don't remove this one, it's important for CI, you can add comment or something: type-tests global